deegree-core/deegree-core-sqldialect/deegree-sqldialect-commons/src/main/java/org/deegree/sqldialect/AbstractSQLDialect.java

```diff
@@ -60,4 +69,16 @@ public boolean isRowLimitingCapable() {
 		return true;
 	}
 
+	@Override
+	public String getSelectBBox(List<String> columns, List<TableName> tables) {
+		if (columns.size() > 1 || tables.size() > 1)
+			LOG.warn("Multiple geometry columns are currently not supported for all SQL dialects or "
+					+ "tunable deegree.sqldialect.consider-all-geometry-columns is false. Using first.");
```

Comment:
This could be misleading in an instance where PostgreSQL and another database is used with the feature enabled.